tests/cases/conformance/classes/members/privateNames/privateNameJsBadAssignment.ts

```diff
@@ -0,0 +1,18 @@
+// @allowJs: true
+// @checkJs: true
+// @noEmit: true
+// @Filename: privateNameJsBadAssignment.js
+
+exports.#nope = 1;           // Error (outside class body)
+function A() { }
+A.prototype.#no = 2;         // Error (outside class body)
+
+class B {}
+B.#foo = 3;                  // Error (outside class body)
+
+class C {
+    #bar = 6;
```

Comment:
this previously included an example like:

```js
class C {
     constructor () {
          exports.#bar = 3;
      }
}
```

The example didn't error when run in the test runner with `gulp runtests`.
It did error (as desired) when run with `gulp runtests -t=privateNameJsBadAssignment` and it did error (as desired) via `tsc` and the language service. I think we found a very minor bug in the test harness.